sniffles/sniffles.py

```diff
@@ -191,6 +206,8 @@ def start_generation(sconf):
                 # Update the content with saved information
                 bt_rule = BackgroundTrafficRule()
                 bt_rule.updateContent(None, back_dist_list, back_absent_proto)
+                # Add the application protocol to the rule name
+                btrule.setRuleName("Background Traffic-" + bt_rule.getProtocolType())
```

Comment:
This is separate from the functionality I want to add. But with this code, I can measure the success  approximatly.